chia/rpc/wallet_rpc_api.py

```diff
@@ -3254,3 +3264,174 @@ async def dl_delete_mirror(self, request) -> Dict:
+            if WalletType(wallet.type()) == WalletType.VC:
+                assert isinstance(wallet, VCWallet)
+                vc_wallet: VCWallet = wallet
+                break
+        else:
+            # Create a new VC wallet
+            vc_wallet = await VCWallet.create_new_vc_wallet(
+                self.service.wallet_state_manager, self.service.wallet_state_manager.main_wallet
+            )
+        vc_record, tx_list = await vc_wallet.launch_new_vc(did_id, puzhash, uint64(request.get("fee", 0)))
+        for tx in tx_list:
+            await self.service.wallet_state_manager.add_pending_transaction(tx)
+        return {
+            "vc_record": vc_record.to_json_dict(),
+            "transactions": [tx.to_json_dict_convenience(self.service.config) for tx in tx_list],
+        }
+
+    async def vc_get_vc(self, request) -> Dict:
+        """
+        Given a launcher ID get the verified credential
+        :param request:
+        :return:
+        """
+        vc_record = await self.service.wallet_state_manager.vc_store.get_vc_record(
+            bytes32.from_hexstr(request["vc_id"])
+        )
+        return {"vc_record": vc_record}
+
+    async def vc_get_vc_list(self, request) -> Dict:
+        """
+        Get a list of verified credentials
+        :param request:
+        :return:
+        """
+        vc_list = await self.service.wallet_state_manager.vc_store.get_vc_record_list(
+            request.get("start", 0), request.get("count", 50)
+        )
+        return {
+            "vc_records": [{"coin_id": "0x" + vc.vc.coin.name().hex(), **vc.to_json_dict()} for vc in vc_list],
+            "proofs": {
+                rec.vc.proof_hash.hex(): None if fetched_proof is None else fetched_proof.key_value_pairs
+                for rec in vc_list
+                if rec.vc.proof_hash is not None
+                for fetched_proof in (
+                    await self.service.wallet_state_manager.vc_store.get_proofs_for_root(rec.vc.proof_hash),
+                )
+            },
+        }
+
+    async def vc_spend_vc(self, request) -> Dict:
+        """
+        Spend a verified credential
+        :param request:
+        :return:
+        """
+        vc_id: bytes32 = bytes32.from_hexstr(request["vc_id"])
+        new_puzhash: Optional[bytes32] = None
+        if "new_puzhash" in request and request["new_puzhash"] is not None:
+            new_puzhash = bytes32.from_hexstr(request["new_puzhash"])
+
+        new_proof_hash: Optional[bytes32] = None
+        if "new_proof_hash" in request and request["new_proof_hash"] is not None:
+            new_proof_hash = bytes32.from_hexstr(request["new_proof_hash"])
+
+        provider_inner_puzhash: Optional[bytes32] = None
+        if "provider_inner_puzhash" in request and request["provider_inner_puzhash"] is not None:
+            provider_inner_puzhash = bytes32.from_hexstr(request["provider_inner_puzhash"])
```

Comment:
This applies to all RPC endpoints.. 

- Don't you think a `vc` prefix is enough and we don't need also a postfix? Like `vc_get_vc` -> `vc_get` or `vc_get_vc_list` -> `vc_get_list` and `vc_spend_vc` -> `vc_spend`
- If you specify the request as streamable schema like below you can avoid repeated checks and manual parsing
```python
@streamable
class VCSpend(Streamable):
    vc_id: Optional[bytes32] = None
    new_proof_hash: Optional[bytes32] = None
    provider_inner_puzhash: Optional[bytes32] = None
    fee: uint64 = uint64(0)
    reuse_puzhash: bool = False
```

and you can just do 

```python
vs_spend = VCSpend.from_json_dict(request)
```

Reply:
suggestion taken